docs/examples/prototypes.md

```diff
@@ -65,26 +65,152 @@ amadeus-travel-restrictions-node ([Link to GitHub](https://github.com/amadeus4de
+
+
+| Use case                                   | Amadeus APIs used                                                                                 | Technology           | Details                                                                                                          |
+|--------------------------------------------|---------------------------------------------------------------------------------------------------|----------------------|------------------------------------------------------------------------------------------------------------------|
+| Trip purpose prediction                    | Trip Purpose Prediction                                                                           | Python, django       | [amadeus-trip-purpose-django](#amadeus-trip-purpose-django)                                                                                      |
+| Hotel Search                               | Hotel Search                                                                                      | Swift                | [amadeus-hotel-search-swift](#amadeus-hotel-search-swift)                                                                                       |
+| Hotel booking engine                       | Hotel Search, Hotel Booking                                                                       | Kotlin               | [amadeus-hotel-booking-android](#amadeus-hotel-booking-android)                                                                                    |
+| Flight Search with Artificial intelligence | Flight Offers Search, Flight Choice Prediction, Trip Purpose Prediction and Airport & City Search | Python, django       | [amadeus-smart-flight-search-django](#amadeus-smart-flight-search-django)                                                                              |
+| Flight Search                              | Flight Offers Search                                                                              | PHP, wordpress       | [amadeus-flight-search-wordpress-plugin](#amadeus-flight-search-wordpress-plugin)                                                                           |
+| Flight Booking engine                      | Flight Offers Search, Flight Offers price, Flight Create Orders, Airport & City Search             | Java, React          | [amadeus_java_flight_api](#amadeus_java_flight_api)                                                                                          |
+| Airport & City autocomplete                | Airport & City Search                                                                             | Node, express, React | [amadeus-airport-city-search-mern](#amadeus-airport-city-search-mern)                                                                                 |
```

Comment:
Can we write this as "Express" instead of "express"? so it will be the same as  the prototype in line 94 and the convention https://expressjs.com/